### PR TITLE
Workbook for SQL VM Assessment

### PR DESCRIPTION
**Pull request by qingquanxu**
## Overview

Add a workbook template for SQL VM Assessment results. This workbook will be a read-only workbook ONLY utilized through the SQL VM resource “SQL Assessment ” tab.

## Screenshot of SQL VM Assessment Workbook

new workbook

### CODE REVIEW

File: Workbooks/Azure SQL VM/SQL Assessment/SQL Assessment.workbook [hunk 1]
```diff
@@ -0,0 +1,661 @@
...
+                    "version": "KqlParameterItem/1.0",
+                    "parameters": [
+                      {
+                        "id": "ca3f7f64-12c1-49c9-b5bc-b29ba33e4d45",
+                        "version": "KqlParameterItem/1.0",
+                        "name": "AsmtId",
+                        "label": " collected at",
+                        "type": 2,
+                        "isRequired": true,
+                        "query": "SqlAsmtLogs_CL\r\n| where TimeGenerated > ago(180d) and _ResourceId =~ {VmResourceId:value}\r\n| extend asmt = parse_csv(RawData)\r\n| project AsmtId = tostring(asmt[1]), TimeGenerated\r\n| summarize AsmtTime = max(TimeGenerated) by AsmtId\r\n| order by AsmtTime desc\r\n| extend rn=row_number()\r\n| project value = AsmtId, label = strcat(format_datetime(AsmtTime, 'yyyy-MM-dd hh:mm tt'), ' UTC'), selected = rn == 1",
```
Comment (vadeveka) [inline Workbooks/Azure SQL VM/SQL Assessment/SQL Assessment.workbook:82]: NIT: Switch to SqlAssessment_CL,  asmt wont be clear to advanced users reviewing Log Analytics configuration

**Review by naxing123 — APPROVED** [review]

File: Workbooks/Azure SQL VM/SQL Assessment/SQL Assessment.workbook (hunk 1 above)
Comment (naxing123) [reply Workbooks/Azure SQL VM/SQL Assessment/SQL Assessment.workbook:82]: are you suggesting replacing all SqlAsmtLogs_CL with SqlAssessment_CL in the workbook? I don't think SqlAssessment_CL will be recognized, Qingquan just tried and it doesn't work. What else should we modify to support SqlAssessment_CL?

Comment (vadeveka) [reply Workbooks/Azure SQL VM/SQL Assessment/SQL Assessment.workbook:82]: The custom log definition would also need to be updated to `SqlAssessment` (_CL is automatically added by LA)

image

Comment (qingquanxu) [reply Workbooks/Azure SQL VM/SQL Assessment/SQL Assessment.workbook]: I've replaced all SqlAsmtLogs_CL with SqlAssessment_CL, should be good now

**Review by vadeveka — APPROVED** [review]

**Review by gardnerjr — APPROVED** [review]
i only verified the codeowners changes, i did not verify the content or behaviors of the given workbook templates

**Comment by gardnerjr** [comment]
also, there were no gallery additions/changes to show this template?  is it only referenced directly from code?

**Comment by naxing123** [comment]
> also, there were no gallery additions/changes to show this template? is it only referenced directly from code?

Correct, we will only reference this workbook from portal code.

**Pull request merged**